Update configure.sh

THX723 opened the pull request:
Made `pi-bluetooth` install for Raspberry Pi only.

THX723 committed:
Update configure.sh

Made pi-bluetooth install for Raspberry Pi only.

marcone commented on setup/pi/configure.sh:
```diff
@@ -250,18 +250,24 @@ function check_and_configure_tesla_ble () {
 
-    if dpkg-query -W --showformat='${db:Status-Status}\n' "pi-bluetooth" 2>/dev/null | grep -q '^installed$'
+    # Install pi-bluetooth for Raspberry Pi only
+    if grep -q "Raspberry Pi" /proc/cpuinfo
```
This checks the hardware version (and a better way to do that would be to check /sys/firmware/devicetree/base/model), which isn't necessarily indicative of a given package existing. 
I think a more correct way to check if a package is available would be to do something like `apt-cache search pi-bluetooth`.

THX723 replied:
Done. Should this check be extended to all other package installs too?

marcone reviewed (commented):
I thought I had already mentioned in a previous review that pi-bluetooth is Raspberry Pi OS specific, but I guess that got lost in the various reworks.

THX723 committed:
Update configure.sh

Check if pkg exists for given hardware, instead of hard check for Raspberry Pi.

THX723 commented:
> I thought I had already mentioned in a previous review that pi-bluetooth is Raspberry Pi OS specific, but I guess that got lost in the various reworks.

Slipped through the crack. Should the `apt-cache search` check be extended to all other package installs too?

marcone commented:
> Should the `apt-cache search` check be extended to all other package installs too?

I don't think it's necessary to do this for packages in general. Most packages are generic and available for every device and Linux distribution.
